nuntium/user_section/views.py

```diff
@@ -621,7 +650,32 @@ def get_redirect_url(self, *args, **kwargs):
             view_messages.info(self.request, _("This message has been marked as public"))
         else:
             view_messages.info(self.request, _("This message has been marked as private"))
-        return reverse('messages_per_writeitinstance', subdomain=self.request.subdomain)
+        return reverse('messages_per_writeitinstance')
```

Comment:
why? sounds like a bad idea. Sending people to the correct subdomain is important.

Reply:
Bad cut and paste. Broke tests and everything.